Sources/TuistSigning/SigningCipher.swift

```diff
@@ -168,7 +168,7 @@ public final class SigningCipher: SigningCiphering {
         let unencryptedData = try FileHandler.shared.readFile(unencryptedFile)
         let encryptedBase64String = try aesCipher.encrypt(unencryptedData.bytes).toBase64()
         guard
-            let data = (iv.base64EncodedString() + "-" + encryptedBase64String).data(using: .utf8)
+            let data = (iv.base64EncodedString() + "-" + encryptedBase64String!).data(using: .utf8)
```

Comment:
this is not necessary if you use `1.4.1` of `CryptoSwift`, so I'd suggest doing that (this change is also why main is breaking, it's probably related to inconsistencies between the `Package.resolved` or `.package.resolved`)